integration-tests/hibernate-orm-panache/src/test/java/io/quarkus/it/panache/PanacheMockingTest.java

```diff
@@ -25,6 +25,40 @@
     public void testPanacheMocking() {
         PanacheMock.mock(Person.class);
 
+        // does not throw (defaults to doNothing)
+        Person.voidMethod();
+
+        // make it throw our exception
+        Mockito.doThrow(new RuntimeException("Stef")).when(PanacheMock.getMock(Person.class)).voidMethod();
+        try {
+            Person.voidMethod();
+            Assertions.fail();
+        } catch (RuntimeException x) {
+            Assertions.assertEquals("Stef", x.getMessage());
+        }
```

Comment:
You can also use the AssertJ API for improved readability: 

```suggestion
        assertThatExceptionOfType(RuntimeException.class)
                .isThrownBy(Person::voidMethod)
                .withMessage("Stef");
```

Reply:
Given that static method references don't work the same as normal static method calls, I'd rather not introduce this style which can lead to surprises.